test.py

```diff
@@ -31,8 +31,7 @@ def test_2d(self):
         res = self.fut.test5(data)
         pyres = self.fut.from_futhark(res)
         assert_array_equal(pyres, data*2)
-        with self.assertRaises(ValueError):
-            self.fut.test5(data.T)
+        self.fut.test5(data.T)
```

Comment:
There should probably be an assertion here to show that the transformed one worked?

Reply:
Agreed.  Will add.